### DISCUSSION TITLE
Show only Your Squadron Missions?

### DISCUSSION THREAD

**Discussion by GrEaSeLiTeNiN** [Khopa/dcs_liberation#1739] [Q&A] [answered]
Hi, is  there a way to belong to say an F18 squadron and have the option to see only Hornet missions? Much like the BMS dynamic campaign where you belong to an F16 squadron and all missions generated for your pick involve only F16s. Of course you can then switch squadrons to fly other specific aircraft if you like. 

At the moment, the list of Liberation missions we get to choose from include a mix of aircraft types and missions. I have to filter through them to find F18s to fly. I understand DCS has many modules to fly and some like it that way. But I'd like the option to focus only on one or two, like the F18s and F16s. Is that possible with Liberation 5?

Thanks!

**Comment by DanAlbert** [comment]
You can set a number of pilots in the squadron to be players and then use this to make it so that the player pilots are preferred for that squadron's missions:
![image](https://user-images.githubusercontent.com/315852/141666150-1ae3c797-c4d7-470c-8da6-615054bbf80c.png)

**Comment by GrEaSeLiTeNiN** [comment, marked as answer]
Update: Thanks. So I change them to "Player" in the UI Air Wing tab? If I do that, yes, the package list shows more "Player Slots". But if for example, I change all 12 pilots to "Player" in an F16 squadron in the Air Wing tab, would that mean that while I am in the cockpit, none of the other F16s in my squadron will be taking off for their own mission because they are all "Players" waiting for me to fly them?

**Reply by DanAlbert** [reply]
Of course. Don't make more player pilots than you have players.

**Reply by GrEaSeLiTeNiN** [reply]
Yeah I had tested it earlier with a 2 ship to confirm. That is, Player Slots are not auto fielded by AI if you do not use them at mission launch. They just don't appear in the mission, like my wingman was missing at the cat launch. But then that brings me back to square one (apart from the prefer player pilots option). So actually there really isn't a way to do what I was asking for in the OP.

**Reply by DanAlbert** [reply]
The option causes player pilots to be preferred _if the planner chooses the squadron_. That's all there is right now. If your hornet squadron is used, the player pilots get used. If it's not getting used, it's not a very useful squadron.
